File: java_backup/my java/ronak/sum5.java
import java.io.*;
class sum5
{
public static void main(int n)
{
int i,j;
char ch;
for(i=1;i<=n;i++)
{
if(i%2==0)
ch='*';
else
ch='#';
for(j=1;j<=i;j++)
{
System.out.print(ch);
if(ch=='#')
ch='*';
else
ch='#';
}
System.out.println("");
}
}
}

File: 6th_sem/MATLABCIA/MATLAB_ANUBHAV/sum5.m
%function s=sum5(n) : Function to calculate s=(1)+(1+2)+...+(1+2+...+n)
function s=sum5(n)
s=0;
f=1;
  for i=1:n
    s=s+f*i;
    f=f*-1;
  end
end
